UnityProject/Assets/Scripts/Objects/Singularity.cs

```diff
@@ -243,7 +244,7 @@ private void SingularityUpdate()
 				Vector3Int EMPPosition = registerTile.WorldPositionServer;
 				EMPPosition.x += UnityEngine.Random.Range(-3 - (int)CurrentStage, 3 + (int)CurrentStage);
 				EMPPosition.y += UnityEngine.Random.Range(-3 - (int)CurrentStage, 3 + (int)CurrentStage);
-				Explosion.StartExplosion(EMPPosition, EMPStrength, true);
+				Explosion.StartExplosion(EMPPosition, EMPStrength, new ExplosionEmpNode());
```

Comment:
why new node?

Reply:
I tried passing just the type of explosion node, like ExplosionEmpNode.GetType() but c# didnt like it